### ISSUE TITLE
Issue with require("fs")

### ISSUE THREAD

**Issue by MultiQueries** [electron/electron#21695]
Hi,
I cannot use the line:
require("fs")
I tried the advice of downgrading glup, downgrading node, upgrading glup, upgrading node js.
I tried the advice of adding the json file that overrides glup.

If I remove the require("fs") then it will work.

So node@10 + glup 3.9.1 fails, node@latest + glup 3.9.1 fails, node@10 + glup 4.0.2 fails,
node@latest + gulp 4.0.2 fails all with the same error as bellow.

I am on a Macbook Pro under Mac OSX Catalina 10.15.2.

The error I receive is as follows:
> electron .

App threw an error during load
ReferenceError: primordials is not defined
    at fs.js:27:26
    at req_ (/Users/uselesspath/path/path/path/path/projectitle/node_modules/natives/index.js:143:24)
    at Object.req [as require] (/Users/uselesspath/path/path/path/path/projectitle/node_modules/natives/index.js:55:10)
    at Object.<anonymous> (/Users/uselesspath/path/path/path/path/projectitle/node_modules/graceful-fs/fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:880:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:892:10)
    at Module.load (internal/modules/cjs/loader.js:735:32)
    at Module._load (internal/modules/cjs/loader.js:648:12)
    at Module._load (electron/js2c/asar.js:717:26)
    at Function.Module._load (electron/js2c/asar.js:717:26)

**Comment by miniak** [comment]
Thanks for reaching out!

We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.

Thanks for understanding and meeting us half way :grinning:

**Comment by MultiQueries** [comment]
Let me try again then.

**Comment by MultiQueries** [comment]
Sorry, the problem is not require("fs")
the problem is 
require("unzip")
apparently NodeJs does not support the required internals anymore.
I feel stupid. I will destroy this comment. I don't know why I believed StackOverflow from 2017.
Sorry again.
